plugins/tiddlywiki/menubar/menu.tid

```diff
@@ -7,34 +7,37 @@ tags: $:/tags/PageTemplate
 <$list filter="[<currentTiddler>addprefix[$:/config/plugins/menubar/MenuItems/Visibility/]get[text]] ~show +[match[show]]" variable="ignore">
 <$list filter="[[$size$]match[wide]] ~[<currentTiddler>get[show-when]match[$size$]] ~[{$:/state/popup/menubar/hamburger}match[yes]]" variable="ignore">
 <li style={{!!custom-menu-styles-$size$}} class={{{ [<currentTiddler>get[show-when]addprefix[tc-menubar-]] tc-menubar-item +[join[ ]] }}}>
-<$list filter="[<currentTiddler>!is-dropdown[yes]]" variable="listItem" emptyMessage="""
+<%if [<currentTiddler>!is-dropdown[yes]] %>
+<%if [<currentTiddler>has[custom-menu-content]] %>
+	<$transclude field="custom-menu-content" mode="inline"/>
+<%else%>
+	<!-- Link -->
```

Comment:
IMO this comment id redundant, since the next widget is `$link`

Reply:
I was mostly preserving the existing code, but I agree that it's redundant.